frameworks/FSharp/giraffe/src/App/Stock.fs

```diff
@@ -34,6 +34,6 @@ let application : HttpHandler =
 
     choose [
         route "/plaintext" >=> text "Hello, World!" 
-        route "/json" >=> json { JsonMessage.message = "Hello, World!" }
+        route "/json" >=> json {| message = "Hello, World!" |}

```

Comment:
should this also be struct!?

Reply:
Potentially, these changes were just me converting like-for-like. I think the stock app is designed to be as plain as possible, so I think a regular record is appropriate.